src/main/java/org/kainos/ea/model/JobRole.java

```diff
@@ -25,8 +48,31 @@ public void setName(String name) {
         this.name = name;
     }
 
+    public int getBandId() {
+        return bandId;
+    }
+
+    public void setBandId(int bandId) {
+        this.bandId = bandId;
+    }
+
+    public String getSpecificationDesc() {
+        return specificationDesc;
+    }
+
+    public void setSpecificationDesc(String specificationDesc) {
+        this.specificationDesc = specificationDesc;
+    }
```

Comment:
Any chance this branch was created off the us2 branch? There's some specification stuff showing as new here that looks like it's been added as part of that story

Reply:
Yea it is because of my ticket falling the latest i felt it would need to have both specification and capability to avoid any merge conflicts